test/parallel/test-performance.js

```diff
@@ -7,6 +7,10 @@ const { performance } = require('perf_hooks');
 assert(performance);
 assert(performance.nodeTiming);
 assert.strictEqual(typeof performance.timeOrigin, 'number');
+assert(Math.abs(performance.timeOrigin - Date.now()) < 300);
+
+const inited = performance.now();
+assert(inited < 300);
```

Comment:
This has me just slightly concerned since it's somewhat arbitrary and the test is in parallel. Anything that can be done about that? I'm guessing not but figured would check...

Reply:
Not much, though this indeed failed on some Windows CIs. More relaxed bounds incoming.